azure-pipelines-wrapper/conflict_detect.js

```diff
@@ -0,0 +1,79 @@
+const akv = require('./keyvault');
+const spawnSync = require('child_process').spawnSync;
+const owner = 'sonic-net'
+const repo = 'sonic-buildimage'
+
+function init(app) {
+    app.log.info("Init conflict detect");
+
+    app.on(["pull_request.opened", "pull_request.synchronize", "pull_request.reopened", "issue_comment.created"], async (context) => {
```

Comment:
Only take action for the issue_comment on the pull request, right? What is the action when the issue comment on an issue?

Reply:
It will fail on git fetch +refs/pull/{id}/merge:remotes/pull/{id}/merge.
Then nothing happened.